src/styles/services-view.less

```diff
@@ -10,6 +10,14 @@ services-view,
 }
 
 .services-view {
+  .pf-empty-state {
```

Comment:
Nit:  we alphabetize rules by selector, so this should go after .blank-slate-pf

Reply:
Okie dok will do!